Gemfile

```diff
@@ -173,3 +173,5 @@ gem 'net-ssh'
 gem 'net-scp'
 gem 'httparty'
 gem 'jquery-cookie-rails'
+gem 'oj'
```

Comment:
Yep, I'm using Oj for it's stricter serialization.  I definitely want an error to be raised rather than #<object:id> string that the JSON module generates.  

I agree that adding another json library is non ideal but that standard library has no way of changing it's behaviour. It's either Oj or manually validating the object which adds a bunch of duplicate work.